pontoon/sync/core/translations_to_repo.py

```diff
@@ -223,3 +240,151 @@ def update_changed_resources(
                 log.error(f"{lc_scope} Update failed: {error}")
                 continue
     return count, updated_locales, translators
+
+
+def set_translations(
```

Comment:
Once the new data model is in place, I assume this function will be much simpler.